app/jobs/email.py

```diff
@@ -43,3 +43,41 @@ def send_confirm_email(email_address, token_url):
         server.close()
     except Exception as e:
         raise (e)
+
+
+@Q.job(timeout=60000)
+def send_password_change_confirm_email(email_address):
+    smtp_username = current_app.config["MAIL_USERNAME"]
```

Comment:
I'm curious - if these things are unlikely to change is there a reason to call them in the message body rather than at the top level?

Reply:
They need current_app to read the vars and it isnt available until after startup

Reply:
Is the top level not read in on startup?